app/models/build.rb

```diff
@@ -7,7 +7,8 @@ class Build < ActiveRecord::Base
   belongs_to :project, inverse_of: :builds
   belongs_to :docker_build_job, class_name: 'Job', optional: true
   belongs_to :creator, class_name: 'User', foreign_key: 'created_by', inverse_of: :builds
-  has_many :deploys, dependent: nil
+  has_many :deploy_builds, dependent: nil
```

Comment:
no reason to leave join associations that don't work 🤷‍♂ 
```suggestion
  has_many :deploy_builds, dependent: :destroy
```